### ISSUE TITLE
ecs_cluster destroy failed - "ClusterContainsContainerInstancesException"

### ISSUE THREAD

**Issue by pikeas** [hashicorp/terraform#2691]
`terraform delete` throws:

```
1 error(s) occurred:

* ClusterContainsContainerInstancesException: The Cluster cannot be deleted while Container Instances are active.
	status code: 400, request id: []
```

I've deleted and recreated this stack a few times, and this is the first time I've seen this error. Shouldn't Terraform handle the dependency instead of throwing this error?

Terraform v0.6.1-dev (ab0a7d8096df5ef45e3d7fa995cccf399220264f)

**Comment by radeksimko** [comment]
Thanks for the report, #2698 addresses that issue.

**Comment by radeksimko** [comment]
Fixed via #2698

**Comment by josh-padnick** [comment]
FYI, I just received this error myself. Running Terraform 0.6.16. This is the first time I've seen it so I'm assuming it's intermittent.
